### PR TITLE
remove unit test add RSpec

### PR DESCRIPTION
**Pull request by AliTopal89**
`rm -rf test` folder
```
echo 'gem "rspec-rails", :group => [:development, :test]' >> Gemfile
bundle install
rails g rspec:install
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 log/
 tmp/
 .powenv
+.rspec
```
Comment (osondoar) [inline .gitignore]: what if we make `.rspec` part of the repo? I usually just have it set up to display colors. You can do that by adding this:


```
--color

```

**Comment by osondoar** [comment]
👍 Just added a tiny comment about adding `.rspec` to the repo.

File: .gitignore (hunk 1 above)
Comment (iamanissa) [reply .gitignore]: I agree it shouldn't be ignored. If you want a different set up you can set an rpsec (global) for your personal computer that will override this one
http://stackoverflow.com/questions/1819614/how-do-i-globally-configure-rspec-to-keep-the-color-and-format-specdoc-o

**Comment by AliTopal89** [comment]
oh yes you are right my bad I will change that

**Review by jmkoni — APPROVED** [review]
👍

**Pull request merged**